lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -304,8 +292,8 @@ function dosomething_northstar_log_request($op, $user, $request_body, $response)
       'user_country' => $user->field_address[LANGUAGE_NONE][0]['country'],
       'fastly_country' => dosomething_settings_get_geo_country_code(),
       'request_values' => json_encode($request_body),
-      'response_code' => $response->code,
-      'response_values' => $response->data,
+      'response_code' => (isset($response['code']) ? $response['code'] : '200'),
```

Comment:
yeah, this was a bit weird. I had done this while I still had  some of the functions using the old approach, but now that I've updated the lot, can probs do something else. I'll see what I can come up with based on your suggestion. The error message has a code, I can try json_decoding it and then grabbing it for this log?